launchable/commands/record/tests.py

```diff
@@ -241,7 +241,7 @@ def tests(
             raise e
         else:
             traceback.print_exc()
-            return
+            exit(0)
```

Comment:
```suggestion
            # To prevent users from stopping the CI pipeline, we exit with a status code of 0, indicating that the program terminated successfully.
            exit(0)
```